packages/frontend/src/components/MkEventEditor.vue

```diff
@@ -0,0 +1,272 @@
+const startTime = ref('00:00');
+const endDate = ref('');
+const endTime = ref('');
+const location = ref(props.modelValue?.metadata.location ?? null);
+const url = ref(props.modelValue?.metadata.url ?? null);
+const showAdvanced = ref(false);
+const doorTime = ref(props.modelValue?.metadata.doorTime ?? null);
+const organizer = ref(props.modelValue?.metadata.organizer?.name ?? null);
+const organizerLink = ref(props.modelValue?.metadata.organizer?.sameAs ?? null);
+const audience = ref(props.modelValue?.metadata.audience?.name ?? null);
+const language = ref(props.modelValue?.metadata.inLanguage ?? null);
+const ageRange = ref(props.modelValue?.metadata.typicalAgeRange ?? null);
+const ticketsUrl = ref(props.modelValue?.metadata.offers?.url ?? null);
+const isFree = ref(props.modelValue?.metadata.isAccessibleForFree ?? false);
+const price = ref(props.modelValue?.metadata.offers?.price ?? null);
+const availabilityStart = ref(props.modelValue?.metadata.offers?.availabilityStarts ?? null);
+const availabilityEnd = ref(props.modelValue?.metadata.offers?.availabilityEnds ?? null);
+const keywords = ref(props.modelValue?.metadata.keywords ?? null);
+
+function get(): misskey.entities.Note['event'] {
+	const calcAt = (date: Ref<string>, time: Ref<string>): number => (new Date(`${date.value} ${time.value}`)).getTime();
+
+	const start = calcAt(startDate, startTime);
+	const end = endDate.value ? calcAt(endDate, endTime) : null;
+	return {
+		title: title.value,
+		start: start,
+		end: end,
+		metadata: {
+			'@type': 'Event',
+			name: title.value,
+			startDate: (new Date(start)).toISOString(),
+			endDate: end ? (new Date(end)).toISOString() : undefined,
+			location: location.value ?? undefined,
+			url: url.value ?? undefined,
+			doorTime: doorTime.value ?? undefined,
+			organizer: organizer.value ? {
+				'@type': 'Thing',
+				name: organizer.value,
+				sameAs: organizerLink.value ?? undefined,
+			} : undefined,
+			audience: audience.value ? {
+				'@type': 'Audience',
+				name: audience.value,
+			} : undefined,
+			inLanguage: language.value ?? undefined,
+			typicalAgeRange: ageRange.value ?? undefined,
+			isAccessibleForFree: isFree,
+			offers: ticketsUrl.value || price.value ? {
+				price: price.value ?? undefined,
+				priceCurrency: undefined,
+				availabilityStarts: availabilityStart.value ?? undefined,
+				availabilityEnds: availabilityEnd.value ?? undefined,
+				url: ticketsUrl.value ?? undefined,
+			} : undefined,
+			keywords: keywords.value ?? undefined,
+		},
+	};
+}
+
+watch([title, startDate, startTime, endDate, endTime, location, url, doorTime, organizer, organizerLink, audience, language, 
+							ageRange, ticketsUrl, isFree, price, availabilityStart, availabilityEnd, keywords], () => emit('update:modelValue', get()), {
+	deep: true,
+});
+</script>
+
+<style lang="scss" scoped>
+.zmdxowut {
+	padding: 8px 16px;
+
+	>.caution {
+		margin: 0 0 8px 0;
+		font-size: 0.8em;
+		color: #f00;
```

Comment:
This should likely `var(--warn)`

Reply:
These styles aren't even needed. Just dead code.